### ISSUE TITLE
Whitelist websites stopped by parental service

### ISSUE THREAD

**Issue by peytonjol** [AdguardTeam/AdGuardHome#3062]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X ] I am running the latest version
- [ X] I checked the documentation and found no answer
- [ X] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * 
* **How did you install AdGuard Home:**
  * 
* **How did you setup DNS configuration:**
  * 
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * 
* **Operating system and version:**
  * 

### Expected Behavior
when i unlock adult website (torproject for instance which is for me not a adult website) it doesn't allow the webiste. Sometime it works but generaly speaking it stays blocked

### Additional Information

How could we bypass adult or securty blocked website ? thanks.

**Comment by ainar-g** [comment]
Hello.  We can't reproduce the issue.  If you have parental protection enabled, you should still be able to unblock certain hosts using custom filtering rules like:

```
@@|example.com^
```

**Comment by peytonjol** [comment]
Already done that before but as i'm testing on linux it works, wasn't on my windows 10. Guess i had some DNS caching prob. Thanks you ! :)
